crates/rolldown/tests/rolldown/misc/preserve_entry_signature/issue-4880/artifacts.snap

```diff
@@ -30,14 +32,15 @@ export { foo_default as default };
 ## foo2.js
 
 ```js
+//! User-defined Entry: [Entry-Module-Id: foo2.js] [Name: Some("foo2")]
 import { t as foo2_default } from "./foo22.js";
 
-export { foo2_default as default };
```

Comment:
This is expected, since our chunk strategy(main.js actually references the `default` from common chunks), the default export of entry `foo2.js` is unused, it is removed since `preserveEntrySignature: false`

Reply:
See `configVariants:{preserveEntrySignature: 'allow-extension'}` result , it is preserved correctly if not `false`